ecl/hql/hqlattr.cpp

```diff
@@ -3868,6 +3868,29 @@ double queryLikelihood(IHqlExpression * expr)
     return likelihoodExpr->queryValue()->getRealValue();
 }
 
+double queryActivityLikelihood(IHqlExpression * expr)
+{
+    double filterLikelihood = 1.0;
```

Comment:
add an assertex(expr->getOperator() == no_filter)

the function should be generalised if it is ever called with something else.